lib/auth/keystore/aws_kms.go

```diff
@@ -54,9 +54,10 @@ import (
 )
 
 const (
-	awskmsPrefix  = "awskms:"
-	clusterTagKey = "TeleportCluster"
-	awsOAEPHash   = crypto.SHA256
+	awskmsPrefix        = "awskms:"
+	clusterTagKey       = "TeleportClusterV2"
+	preventDeleteTagKey = "TeleportPreventDelete"
```

Comment:
I decided to rename these tags to better reflect what they're for. I swapped `TeleportCluster` to `TeleportClusterV2` in order to prevent previous auth service versions from unintentionally removing encryption keys